### DISCUSSION TITLE
Background Color

### DISCUSSION THREAD

**Discussion by billybena** [LianqiangGuan/Tafe230211#15] [General]
What color should we use for the background?

**Comment by LianqiangGuan** [comment]
currently we use alice-blue

**Reply by billybena** [reply]
Thanks Lian. I will make the changes to mine.

**Reply by billybena** [reply]
Is alice-blue the color for all pages? I just want to confirm.

**Reply by LianqiangGuan** [reply]
just for the main menu.
